teuthology/misc.py

```diff
@@ -1168,7 +1168,7 @@ def ssh_keyscan_wait(hostname):
     False otherwise. Try again if ssh-keyscan timesout.
     :param hostname: on which ssh-keyscan is run
     """
-    with safe_while(sleep=6, tries=100,
+    with safe_while(sleep=6, tries=100, _raise=False,
```

Comment:
What is the purpose of seting `_raise` to False here? Won't that change the behavior?

Reply:
the purpose to not throw exception when number of tries exhausted, what is actually expected by the code

Reply:
@smithfarm these are actually a cherry-pick from upstream